### DISCUSSION TITLE
Listen on multiple ports

### DISCUSSION THREAD

**Discussion by t3therdev** [bluenviron/mediamtx#443] [Questions]
## Describe the feature

We are having an issue with users from other countries that are being blocked on the normal ports we are using. Would it be possible to be able to have multiple listening ports for RTSPS or RTSP? So that our app could provide the first port and if it doesn't work go to another port?

**Comment by aler9** [comment]
Hello, in my opinion you can easily overcome this issue by using a firewall; for instance, you can accept users from port 9445 by using iptables:
 
 ```
 iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 9445 -j REDIRECT --to-port 8554
 ```

[a comment by an automated account is not shown]
